### PR TITLE
2 minor fixes in document system

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi.
1. I added error message in the construct of thumbnail generator in case that missing GD extension (PHP extension).

2. adding  empty folder in the sites/documents. without this folder php throws warning error when retrieving file from couchDB and doesn't present the file .
Error screenshot -   
image

Thanks.
Amiel

### CODE REVIEW

File: library/classes/thumbnail/Thumbnail.class.php [hunk 1]
```diff
@@ -32,6 +32,12 @@ class Thumbnail
      */
     public function __construct($max_size = null)
     {
+        //validate that GD extension was enabled
+        if (!extension_loaded('gd'))
+        {
+            die('Abort '.basename(__FILE__).' : Missing DG extensions');
```
Comment (bradymiller) [inline library/classes/thumbnail/Thumbnail.class.php]: Will the user see this error?
(Also recommend sending an error message to the php error log)

**Comment by bradymiller** [comment]
When adding a dummy file to hold a directory, it is better to make a README file in the directory with a quick description of the purpose of the directory.

**Comment by bradymiller** [comment]
Review complete. Just some minor issues. Just let me know when it's ready for another review.

File: library/classes/thumbnail/Thumbnail.class.php (hunk 1 above)
Comment (matrix-amiel) [inline library/classes/thumbnail/Thumbnail.class.php]: Yes. he see it - 
image

**Comment by matrix-amiel** [comment]
Ready for code review.

Thanks

**Comment by bradymiller** [comment]
Thanks for this fix! Looks good and just brought it into the codebase.

**Pull request merged**